asdf/fits_embed.py

```diff
@@ -145,6 +147,13 @@ class AsdfInFits(asdf.AsdfFile):
     """
 
     def __init__(self, hdulist=None, tree=None, **kwargs):
+        warnings.warn(
+            "AsdfInFits has been deprecated and will be removed in "
+            "asdf-3.0. Similar functionality has been added to stdatamodels "
+            "https://github.com/spacetelescope/stdatamodels to load and "
+            "save ASDF in fits files.",
+            AsdfDeprecationWarning,
+        )
```

Comment:
This whole module is being deprecated, not just his class. As such the warning should be at the module level.

Reply:
Thanks. I moved the warning and added notes and deprecations for asdftool extract and remove_hdu